File: Leetcode-Problems/Maximum Number of Balloons.cpp
class Solution
{
public:
    int maxNumberOfBalloons(string text)
    {
        int arr[26] = {0};
        for (char c : text)
        {
            arr[c - 'a']++;
        }
        return min({arr[0], arr[1], arr[11] / 2, arr[13], arr[14] / 2});
    }
};

File: Leetcode-Problems/Maximum Number of Balloons.java
class Solution {
    public int maxNumberOfBalloons(String text) {
        int arr[] = new int[26];
        for (char c : text.toCharArray()) {
            arr[c - 'a']++;
        }
        return Math.min(
                Math.min(arr[1], arr[0]),
                Math.min(
                        Math.min(arr[11] / 2, arr[14] / 2),
                        arr[13]));
    }
}